source/docs/articles/sites/logs.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Log Files on Pantheon  
+description: Use log files to identify errors, track response times, analyze visitors and more on your WordPress or Drupal site.
+keywords: log, access log, nginx access log, nginx log, nginx access, nginx error, nginx error log, php error, php error log, php fpm error, php fpm, php error, php slow, php slow log, pyinofity log, pyinofity, watcher log, watcher
+draft: true
+---
+Log files track and record your site's activity which help you find, debug, and isolate current or potential problems on your site. These logs can also be used to compile statistics on your visitors and referral traffic.
```

Comment:
I think we should remove the 2nd sentence, because any traffic hitting Varnish won't hit the application container's nginx so won't be recorded in nginx-access.log